rcongui/src/components/PlayerInfo/PlayerInfo.js

```diff
@@ -0,0 +1,267 @@
+                                        <Typography>{moment(this.state.sessions[0]?.end).format("ddd Do MMM HH:mm:ss")}</Typography>
+                                    </Grid>
+                                    <Grid item>
+                                        <Typography variant={"h6"}>Total play time</Typography>
+                                        <Typography>{moment.duration(this.state.total_playtime_seconds, "seconds").humanize()}</Typography>
+                                    </Grid>
+                                    <Grid item>
+                                        <Typography variant={"h6"}>Player penalties</Typography>
+                                        <Typography>Perma ban: {this.state.penalty_count.PERMABAN}</Typography>
+                                        <Typography>Temp ban: {this.state.penalty_count.TEMPBAN}</Typography>
+                                        <Typography>Kick: {this.state.penalty_count.KICK}</Typography>
+                                        <Typography>Punish: {this.state.penalty_count.PUNISH}</Typography>
+                                    </Grid>
+                                </Grid>
+                            </Grid>
+
+                            <Grid item sm={9}>
+                                <Grid container spacing={3} justify={"flex-start"} alignItems={"flex-start"}
+                                      alignContent={"flex-start"}>
+                                    <Grid item sm={12}>
+                                        <Grid container justify={"flex-start"} alignItems={"flex-start"}
+                                              alignContent={"flex-start"}>
+                                            <NamePopOver names={this.state.names}/>
+                                        </Grid>
+                                    </Grid>
+                                    <Grid item sm={4}>
+                                        <Grid container sm={12} justify={"space-between"}>
+                                            <Grid item><Typography color={this.state.vip ? "primary" : ""}
+                                                                   variant={"button"}>VIP</Typography></Grid>
+                                            <Grid item><Typography color={this.state.perma ? "error" : ""}
+                                                                   variant={"button"}>PERMABAN</Typography></Grid>
+                                            <Grid item><Typography color={this.state.temp ? "error" : ""}
+                                                                   variant={"button"}>TEMPBAN</Typography></Grid>
+                                            <Grid item><Typography
+                                                color={this.state.blacklist?.is_blacklisted ? "error" : ""}
+                                                variant={"button"}>BLACKLISTED</Typography></Grid>
```

Comment:
So this will always show, even if the guy is not blacklisted? Maybe it's a bit confusing?

Reply:
Previously It was transparent but I removed all the css